pkg/csi/controller.go

```diff
@@ -46,19 +46,21 @@ var (
 	}
 )
 
-
 type controllerServer struct {
 	Driver       *VCDDriver
-	VCDCSIClient *vcdcsiclient.Client
+	VCDCSIClient *vcdcsiclient.DiskManager
+	VAppName     string
 }
 
 // NewControllerService creates a controllerService
-func NewControllerService(driver *VCDDriver, vcdClient *vcdclient.Client) csi.ControllerServer {
+func NewControllerService(driver *VCDDriver, vcdClient *vcdsdk.Client, cluserID string, vAppName string) csi.ControllerServer {
 	return &controllerServer{
-		Driver:        driver,
-		VCDCSIClient:  &vcdcsiclient.Client{
+		Driver: driver,
```

Comment:
can we run a `go fmt ...` and a `go vet ...` on the files in GITROOT so that the formatting are fixed.